test/k8sT/Policies.go

```diff
@@ -869,13 +871,12 @@ EOF`, k, v)
 			res = kubectl.Apply(demoPath)
 			res.ExpectSuccess("unable to apply manifest")
 
-			err := kubectl.WaitforPods(secondNS, "-l zgroup=testapp", helpers.HelperTimeout)
-			Expect(err).To(BeNil(),
-				"testapp pods are not ready after timeout in namspace %q", secondNS)
-
-			err = kubectl.WaitforPods(helpers.DefaultNamespace, "-l zgroup=testapp", helpers.HelperTimeout)
-			Expect(err).To(BeNil(),
-				"testapp pods are not ready after timeout in %q namespace", helpers.DefaultNamespace)
+			ExpectDeployReady(kubectl, secondNS, "app1", helpers.HelperTimeout)
```

Comment:
Can you change these and other string literals to use `helpers.AppN` where "N" is one of {1, 2, 3}?